src/fsharp/FSharp.Core/Testing.FSharp.Core.nuspec

```diff
@@ -15,35 +15,13 @@
         <tags>$tags$</tags>
         <dependencies>
             <group>
-                <dependency id="System.Collections" version="4.0.11" />
```

Comment:
This file isn't used in any build, we can delete it.

Reply:
@brettfo, I wondered about that, when I was making the change.  I made a note to myself to check, and then forgot.  Getting older is a tough thing.